discovery-provider/src/queries/get_attestation.py

```diff
@@ -189,13 +193,18 @@ def verify_discovery_node_exists_on_chain(new_sender_address: str) -> bool:
 
 
 def get_create_sender_attestation(new_sender_address: str) -> Tuple[str, str]:
+    """
+    Returns a owner_wallet, signed_attestation tuple,
+    or throws an error explaining why the sender attestation was
+    not able to be created.
+    """
     is_valid = verify_discovery_node_exists_on_chain(new_sender_address)
     if not is_valid:
         raise Exception(f"Expected {new_sender_address} to be registered on chain")
 
     items = [
         to_bytes(text=ADD_SENDER_MESSAGE_PREFIX),
-        to_bytes(hexstr=REWARDS_MANAGER_PROGRAM),
+        bytes(REWARDS_MANAGER_PROGRAM_ACCOUNT),
```

Comment:
might be worth a comment here explaining this gotcha

Reply:
yeees